e2e-tests/cypress/e2e/journeys/insurance/your-business/broker/validation/are-you-using-a-broker-answer-yes.spec.js

```diff
@@ -76,6 +77,28 @@ context('Insurance - Your business - Broker Page - As an Exporter I want to conf
 
       // TOWN error check
       cy.submitAndAssertFieldErrors(broker[TOWN], null, 2, expectedErrorsCount, BROKER_ERRORS[TOWN].IS_EMPTY);
+
+      // // EMAIL error check
+      cy.submitAndAssertFieldErrors(broker[EMAIL], null, 3, expectedErrorsCount, BROKER_ERRORS[EMAIL].IS_EMPTY);
+    });
+  });
+
+  describe(`when the ${EMAIL} input is in the wrong format`, () => {
```

Comment:
Can we use the same validation checks we use for other email fields? Check out `account-create-your-details-validation-email.spec.js`